### DISCUSSION TITLE
WSL2 zsh 配置订阅出现问题

### DISCUSSION THREAD

**Discussion by XIAOZHUXUEJAVA** [daeuniverse/gg#38] [Q&A]
gg config -w subscription=https://1.amazom...................
zsh: no matches found: subscription=https://1.amaz......

**Comment by mzz2017** [comment]
给加双引号试试看呢

**Comment by XIAOZHUXUEJAVA** [comment]
![image](https://user-images.githubusercontent.com/81864199/225224119-39b73ffc-5096-478b-b17c-3c4faa659663.png)

我使用了gg命令 gitclone相比较于不使用更加慢了，可能是我的节点网速的问题 ；
 还有个问题是我如何判断我已经可以访问外网了 我使用ping youtube.com 还是会丢包
抱歉

**Comment by mzz2017** [comment]
@XIAOZHUXUEJAVA ping是icmp的，不走代理，用curl -i google.com

**Comment by XIAOZHUXUEJAVA** [comment]
安装 gg工具  的docker-ubuntu 容器使用上述命令后结果：
![image](https://user-images.githubusercontent.com/81864199/225228067-aaf3cac3-6625-455d-bb9c-29fba41db31b.png)

未安装gg工具 的WSL2容器使用上述命令后的结果：
![image](https://user-images.githubusercontent.com/81864199/225228361-a95a6359-6b0e-4ab8-b966-0ea887bd781b.png)

windows运行上述命令的结果：
![image](https://user-images.githubusercontent.com/81864199/225228533-ab46d7d4-2a0e-493e-89c2-28efa216197e.png)

不知道是否成功了？大佬帮我看看。
还有有没有git issues的使用规则，我这样询问问题会不会对仓库造成影响？如果有我需要删除吗
“ping是icmp的，不走代理，用curl -i google.com”  这个学到了 感谢

**Comment by mzz2017** [comment]
是成功了，没关系，我稍后将这个 issue 转到 discussion。

**Reply by XIAOZHUXUEJAVA** [reply]
好的，感谢，最后一个问题是：我不使用gg命令下，就会使用默认网络对吧？比如git clone  和 gg git clone, 第一个命令不走代理， 第二个走代理

**Reply by mzz2017** [reply]
> 好的，感谢，最后一个问题是：我不使用gg命令下，就会使用默认网络对吧？比如git clone  和 gg git clone, 第一个命令不走代理， 第二个走代理

是的，不用gg就不走代理
